recipe/meta.yaml

```diff
@@ -10,9 +10,10 @@ source:
   sha256: f9e8d81d0405ba66d181529af42a3354f838c939095ff99930da6aa9cdf6fe46
   patches:
     - 0001-Add-separate-options-to-disable-shared-static-librar.patch
+    - 0002-Fix-pkgconfig-rpath.patch
```

Comment:
Could you please add a comment here explaining that the release after ``1.0.9`` this patch will be no longer necessary?

Reply:
Done!

Reply:
Thanks! :)

Reply:
This is great. In the future, you can use jinja2 if statements to turn patches off if the version is not right. This helps automate things.